### ISSUE TITLE
Check for old Backup

### ISSUE THREAD

**Issue by Spiegel-Leser** [asdfjkl/yahb#9]
This is a wish for one additional feature: I'd like a flag to determine a maximum age x. If the last backup file is older than x, then copy instead of hardlink.

The reason: I feel unsafe if an important file was written to memory only once years ago and is only hardlinked since then. I'd feel better if it's written anew once in a while. Of course this relies on the original file aging better than the old Backup copy. So addtionally it would make sense to verify the versions against each other, but I guess that is a lot more work than my proposal above.

**Comment by asdfjkl** [comment]
I do not think that the real danger (bit-rot) can be averted by re-writing files to a disk. Rather, important backups should be saved among multiple, different disks.

In the next version, I will add an "copy everything, no - hardlinks" option though.

**Comment by Spiegel-Leser** [comment]
I think multiple copies on one disk give a higher chance of an
error-free copy or at least error-free parts that could be congregated
by some other tool. But of course multiple disks is better, yes. Only,
this is not always an option. So I stand by my idea.

Your proposal is ok, it decreases the automation factor a little bit:
I'd have to set-up two scheduled tasks, one for the regular Backups
every week, one for complete copies every year.


Am 29.01.2021 um 11:51 schrieb Dominik Klein:
